spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/DataFlowTemplate.java

```diff
@@ -51,62 +51,67 @@
 	/**
 	 * REST client for stream operations.
 	 */
-	private final StreamOperations streamOperations;
```

Comment:
I am fine with both ways. In fact, there is some more work to be done with #641 along with these things.